indexer/services/socks/src/lib/message-forwarder.ts

```diff
@@ -125,12 +139,12 @@ export class MessageForwarder {
       return;
     }
 
-    const start: number = Date.now();
+    const startForwardMessage: number = Date.now();
     this.forwardMessage(messageToForward);
```

Comment:
This function seems to send out websocket messages in batches for certain websocket topics. Can we pipe the initial timestamp all the way to the `sendMessage` function?

Reply:
Do you mean we should add a message_received_timestamp field to 
```
export interface OutgoingMessage {
  type: OutgoingMessageType;
  connection_id: string;
  message_id: number;
}
```?

Reply:
Nvm, the batches happen at periodic intervals so not necessary to pipe it through.

Does it make more sense to have the stat be `end - Number(message.headers?.message_received_timestamp)` (after calling `forwardMessage`) rather than `start - Number(message.headers?.message_received_timestamp)`?

Reply:
done